tests/new_functional/helpers.js

```diff
@@ -81,6 +81,39 @@ function isViewSectionsVisible() {
   return browser.isExisting(viewSectionsLink) && browser.isVisibleWithinViewport(viewSectionsLink);
 }
 
+function checkCompletedTick(link_name) {
```

Comment:
I think this method can be removed now it has been replaced with `isSectionComplete`?